docs/vertx-runtime/master.adoc

```diff
@@ -34,10 +34,10 @@ include::topics/proc_configuring-your-vertx-application-to-use-agroal.adoc[level
 // MISSIONS
 include::topics/con_missions-and-cloud-native-development-on-openshift.adoc[leveloffset=+1]
 
-[[vertx-available-missions-and-boosters]]
-== Available missions and boosters for {Vertx}
+[[vertx-available-examples]]
+== Available examples for {Vertx}
```

Comment:
Do we really need "Available" here? I think "Example applications for {Vertx}" is enough. What do you think?